_posts/2024-10-24-Scaling-Kubernetes-with-Metal3-on-Fake-Node.md

```diff
@@ -0,0 +1,467 @@
+---
+title: "Scaling Kubernetes with Metal3: Simulating 1000 Clusters with Fake Ironic Agents"
+date: 2024-10-24
+draft: false
+categories: ["metal3", "cluster API", "ironic", "baremetal", "scaling"]
+author: Huy Mai
+---
+
+## Introduction
```

Comment:
I suggest dropping this heading. It is technically directly below the title, so it is two headings on top of each other. That is not a big deal but it also makes the excerpt look strange since there it isn't visible as a heading. It looks like this:

> Introduction If you’ve ever tried scaling out Kubernetes clusters in a bare-metal environment, you’ll know that large-scale testing comes with serious challenges. Most of us don’t have access to enough physical servers—or even virtual machines—to simulate the kinds of large-scale environments we need for stress testing, especially when deploying hundreds...